src/devices/CharacterLcd/samples/LcdConsole.Samples.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
```

Comment:
nit: copyright header (here and everywhere where missing)

Reply:
unresolving since it's still missing

Reply:
Sorry, missed that one.